### PR TITLE
Use ThemeType instead of string

### PR DESCRIPTION
**Pull request by djsmith85**
## Type of change

- [ ] Bug fix
- [ ] New feature development
- [X] Tech debt (refactoring, code cleanup, dependency upgrades, etc)
- [ ] Build/deploy pipeline (DevOps)
- [ ] Other

## Objective
Follow up from work done on: https://github.com/bitwarden/jslib/pull/642

Asana task: https://app.asana.com/0/1201648796371593/1201727539415405/f

## Code changes

- **jslib:** Pull in changes from https://github.com/bitwarden/jslib/pull/642
- **src/app/accounts/settings.component.ts:** Set theme's type to `ThemeType`

## Testing requirements
- Migrating from an older client with a different theme than light (as that is the default) should keep the theme's setting
- Changing and saving a theme after an upgrade should also still work.

## Before you submit

- [X] I have checked for **linting** errors (`npm run lint`) (required)
- [ ] This change requires a **documentation update** (notify the documentation team)
- [ ] This change has particular **deployment requirements** (notify the DevOps team)

### CODE REVIEW

**Review by MGibson1 — APPROVED** [review]
Approving but hold off on merge pending @bitwarden/dept-qa approval to increase test load

**Comment by djsmith85** [comment]
As discussed, removing hold and merging. Thanks @MGibson1 for keeping an eye out on QA load.

**Pull request merged**
